src/main/java/lotto/LottoNumber.java

```diff
@@ -0,0 +1,41 @@
+package lotto;
+
+import java.util.Objects;
+
+public class LottoNumber {
+	public static final String INVALID_NUMBER = "잘못된 숫자입니다.";
+	private final int number;
+
+	LottoNumber(int number) {
+		if(!this.validate(number)) {
+			throw new IllegalArgumentException(INVALID_NUMBER);
+		}
+		this.number = number;
+	}
+
+	private boolean validate(int number) {
+		if(number < LottoRule.LOTTO_MINIMUM_NUMBER || LottoRule.LOTTO_MAXIMUM_NUMBER < number) {
+			return false;
+		}
+		return true;
+	}
```

Comment:
위에 리뷰와 동일합니다.! :-) 
아래처럼 바로 return해주셔도 될 것 같아요! 
```suggestion
			return number >= LottoRule.LOTTO_MINIMUM_NUMBER && LottoRule.LOTTO_MAXIMUM_NUMBER >= number;
```

한가지 더  의견을 드리자면 현재 지난주 당첨 번호와 보너스볼에 45이상 숫자를 입력하면 오류가 나는데요! 
오류가 나는 시점이 보너스 볼까지 입력 후 나오는 오류라서 정확히 어떤 숫자에서 나타나는 오류인지 인지가 잘 되지 않고 있습니다!  확인 부탁드릴께요!